pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_checking-xref-validation-errors.adoc

```diff
@@ -0,0 +1,21 @@
+[id="checking-xref-validation-errors_{context}"]
+
+= Checking xref validation errors
+
+[role="_abstract"]
+Check your content for xref validation errors. Invalid xrefs result in broken link when published. 
+
+.Prerequisites
+
+* The content has been imported to {ProductShortName}.
+* The supported forms of xref have been used.
++
+For information on the supported xref forms, see xref:about-xrefs-validation-in-pantheon_{context}[About xref validation in {ProductShortName}].
```

Comment:
Make sure you replace {context} with whatever you expect {context} to evaluate to.